tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -1,56 +1,56 @@
 from functools import partial
-from typing import List, Any, Tuple
+from typing import List, Any, Tuple, Callable, Sequence
 import numpy as np
 Tensor = Any
 
 def _generate_jitted_eigsh_lanczos(jax):
   """
-  Helper function to generate jitted lanczos function used 
-  in JaxBackend.eigsh_lanczos. The function `jax_lanczos` 
+  Helper function to generate jitted lanczos function used
```

Comment:
Did you remove trailing white space on purpose, or is this from yapfing?

Reply:
I removed it on purpose with a vi script

Reply:
why?

Reply:
Trailing whitespace violates PEP8 and is in general Bad because for example it invisible modifies the behaviour of text editor commands to jump to the end of the line. Was there some reason to keep it?

Reply:
no, just curious. If you want it removed, then it would be great if you could prepare a PR that fixes this for the whole library (just run your script over all files). This way we'll have a cleaner code history